lib/game.rb

```diff
@@ -0,0 +1,58 @@
+require_relative '..\lib\player.rb'
+require_relative '..\lib\board.rb'
+class Game
+  attr_reader :player1, :player2, :board
+  def initialize(player1, player2, board)
+    @player1 = player1
+    @player2 = player2
+    @board = board
+    @turn = 1
+    @result_combinations = [[1, 2, 3], [4, 5, 6], [7, 8, 9],
+                            [1, 4, 7], [2, 5, 8], [3, 6, 9],
+                            [1, 5, 9], [3, 5, 7]]
+  end
+
+  def game_start
+    until (winner? || board.full?)
+      @turn == 1 ? play(player1) : play(player2)
```

Comment:
Upon making :turn as part of the `attr_reader` line you can simply call `turn` and not `@turn`